newIDE/app/src/EventsBasedObjectEditor/EventBasedObjectChildrenEditor.js

```diff
@@ -231,6 +231,8 @@ export default class EventBasedObjectChildrenEditor extends React.Component<
                 // Nothing special to do.
                 onObjectCreated={() => {}}
                 onObjectSelected={this._onObjectSelected}
+                renamedObjectWithContext={null}
+                setRenamedObjectWithContext={() => {}}
```

Comment:
@D8H need your approval on this!

Reply:
From what I see, this is ok at the moment. It will be changed when the feature to rename children in EBOs is implemented.

Reply:
Good, thanks!

Sorry, what is "EBOs" ? 😅